### DISCUSSION TITLE
sender email problem

### DISCUSSION THREAD

**Discussion by ArjunKSukesh** [emailjs-com/help#121] [General]
when i send an email using the emailjs, email  received perfectly but when i checked the "from" email and "to" email in the description it showing same email id. please help me how to change the "from" email

**Comment by xr0master** [comment]
duplicate https://github.com/orgs/emailjs-com/discussions/8

**Comment by ArjunKSukesh** [comment]
but it is showing sender email and receiver email as the same in the email details.

**Reply by xr0master** [reply]
So what's your question?
